plugins/sparse_matrix.h

```diff
@@ -71,6 +71,16 @@ struct BaseEntry {
 struct MatrixWriteTransaction;
 
 struct MatrixReadTransaction {
+
+      struct Stream {       
```

Comment:
They are similar. RowStream's next returns RowNames, this one returns the uint64_t hashes specific to the sparse datasets implementation